modules/messages/src/lib.rs

```diff
@@ -695,9 +695,18 @@ fn ensure_owner_or_root<T: Config<I>, I: Instance>(origin: T::Origin) -> Result<
 	}
 }
 
-/// Ensure that the pallet is in operational mode (not halted).
-fn ensure_operational<T: Config<I>, I: Instance>() -> Result<(), Error<T, I>> {
-	if IsHalted::<I>::get() {
+/// Ensure that the pallet is in normal operational mode.
+fn ensure_normal_operating_mode<T: Config<I>, I: Instance>() -> Result<(), Error<T, I>> {
+	if PalletOperatingMode::<I>::get() != OperatingMode::Normal {
+		Err(Error::<T, I>::Halted)
+	} else {
+		Ok(())
+	}
+}
```

Comment:
```suggestion
	if PalletOperatingMode::<I>::get() == OperatingMode::Normal {
		Ok(())
	} else {
		Err(Error::<T, I>::Halted)
	}
}
```

nit: But it matches the check that the function says it does